docs/src/components/Layout/GlobalNav/GlobalNav.module.scss

```diff
@@ -41,143 +44,187 @@
 
 .dev-center-logo {
   color: var(--amplify-colors-neutral-10);
-  font-size: 1rem;
+  font-size: var(--amplify-font-sizes-large);
   white-space: nowrap;
 }
 
-.nav-container {
-  display: flex;
-  height: var(--dev-center-nav-height);
-  align-items: center;
-  justify-content: space-between;
-}
-
-.mobile-nav-container {
-  display: flex;
-  height: var(--dev-center-nav-height);
-  align-items: stretch;
-  justify-content: space-between;
-}
-
 .nav-links-container {
-  height: var(--dev-center-nav-height);
+  z-index: 200;
+  width: 100%;
+  position: absolute;
+  top: 0;
   align-items: center;
   justify-content: space-between;
   align-items: stretch;
-  padding-block: 0;
-  padding-inline: var(--amplify-space-large);
-}
-
-.mobile-nav-menu-container {
-  z-index: 1000;
-  background-color: var(--amplify-colors-neutral-90);
-  position: absolute;
-  display: flex;
+  height: auto;
   flex-direction: column;
-  width: 100%;
+  padding: 0;
+  background-color: var(--amplify-colors-neutral-90);
+  gap: initial;
+  & .left-nav-links,
+  .right-nav-links {
+    flex-direction: column;
+    gap: initial;
+  }
 }
 
-.mobile-nav-icons-container {
-  margin-left: auto;
-  margin-right: auto;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-
-  svg {
-    display: block;
-    margin: auto;
+.nav-expander {
+  &:hover,
+  :active {
+    border: none;
+    box-shadow: none;
   }
-}
 
-.mobile-nav-menu-items {
-  border-top: 1px solid var(--navbar-border-color);
-  padding-inline: var(--amplify-space-small);
-  height: var(--dev-center-nav-height);
-  width: 100%;
+  &:focus {
+    box-shadow: 0 0 0 2px var(--amplify-colors-neutral-10);
+  }
 }
 
 .nav-menu-item {
-  padding: 0 var(--amplify-space-xs);
   display: flex;
   align-items: center;
   height: 100%;
-  font-size: var(--amplify-font-sizes-small);
+  width: 100%;
+  font-size: var(--amplify-font-sizes-medium);
   text-decoration: none;
   color: var(--amplify-colors-neutral-40) !important;
   font-family: 'Amazon Ember';
   font-weight: 400;
 
   white-space: nowrap;
 
-  &:hover,
-  &:focus {
+  &:hover {
     color: var(--amplify-colors-neutral-10) !important;
 
     .custom-path-stroke {
       stroke: var(--amplify-colors-neutral-10);
     }
   }
 
+  &:focus {
+    box-shadow: 0 0 0 2px var(--amplify-colors-neutral-10);
+  }
+
   .custom-path-stroke {
     stroke: var(--amplify-colors-neutral-40);
   }
 }
 
-.secondary-nav-button {
-  border: 1px var(--navbar-border-color);
-  border-style: solid none;
+.chevron-icon {
+  fill: var(--amplify-colors-neutral-40);
+}
 
-  // Set color for back chevron icon
-  color: var(--amplify-colors-neutral-20);
+.background-overlay {
+  background-color: var(--amplify-colors-overlay-40);
+  z-index: 150;
```

Comment:
where is 150 from?

same for line 52

Reply:
Just an arbitrary value to be larger than the `z-index` inside `header.scss`: https://github.com/aws-amplify/amplify-ui/blob/53b6cee0c3c0771ded6944d176f1c3e739575b58/docs/src/styles/docs/header.scss#L27